interview-questions.md

```diff
@@ -7,60 +7,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Git is a version control on your local machine. It can track source code history. It is open source and it is free!
 
-  Researched answer:
+  Researched answer: Git is a software for tracking changes in any set of files, usually for coordinating work among programmers collaboratively developing source code during software development. 
 
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: Relational operators (>, <, =, ...), equality operators (!==, ===, ==), logical operators (&&, ||)
```

Comment:
Yup, you got it! Just a note that `=` is an assignment operator not a relational.